puppet/chassis.rb

```diff
@@ -160,6 +160,6 @@ def self.install_extension(extension)
 
 		folder = @@dir + '/extensions/' + extension.split('/').last.gsub(/.git$/, '')
 
-		system("git clone %s %s --recursive" % [repo, folder] ) unless File.exist?( folder )
+		system("git clone %s %s --recursive" % [repo, folder.downcase] ) unless File.exist?( folder )
```

Comment:
This should be on the `folder =` line, otherwise `File.exist?` will fail

Reply:
Good catch. I've altered that now. Thanks!